src/web/BL_Python/web/scaffolding/templates/base/{{application_name}}/_app_type.py.j2

```diff
@@ -0,0 +1,3 @@
+from flask import Flask
+
+AppType = Flask
```

Comment:
Newline police, attack!

Reply:
Ah - I wondered why Black didn't catch this, but it's because it's a Jinja2 file. Thanks!